scholia/app/templates/base.html

```diff
@@ -60,7 +60,19 @@
     $.getJSON(url, function (data) {
 	 var item = data.entities["{{ q }}"];
 	 if ('en' in item.labels) {
-	     $("#h1").text(item.labels.en.value);
+	     var title = item.labels.en.value
+	     $("#h1").text(title);
+
+	     {% if request.path.endswith("curation") or request.path.endswith("curation/") %}
+
+	     var linkedToFrom = "<p><a href='https://www.wikidata.org/w/index.php?hidecategorization=1&target={{ q }}&showlinkedto=1&limit=500&days=7&enhanced=1&title=Special:RecentChangesLinked&urlversion=2' >";
+	     linkedToFrom += "Recent changes to items that link <em>to</em> " + title + "</a> and ";
```

Comment:
Shouldn't `title` be escaped, e.g., an article title could have a "<" in it? (and XKCD 327)